File: admin/src/main/java/com/weicoder/admin/po/LogsOperate.scala
package com.weicoder.admin.po

import javax.persistence.Entity
import org.hibernate.annotations.DynamicInsert
import org.hibernate.annotations.DynamicUpdate
import org.springframework.context.annotation.Scope
import org.springframework.stereotype.Component
import org.springframework.beans.factory.config.ConfigurableBeanFactory
import com.weicoder.site.entity.base.BaseEntityIdTime
import com.weicoder.base.entity.EntityUser
import com.weicoder.base.entity.EntityIp
import scala.beans.BeanProperty
import com.weicoder.base.entity.EntityUserId

/**
 * 操作日志实体
 * @author WD
 * @since JDK7
 * @version 1._ 2_11-_4-_3
 */
@Component
@Scope(ConfigurableBeanFactory.SCOPE_PROTOTYPE)
@Entity
@DynamicInsert
@DynamicUpdate
class LogsOperate extends BaseEntityIdTime with EntityUserId with EntityIp {
  // 内容 
  var content: String = _
  // 用户ID 
  @BeanProperty
  var userId: Integer = _
  // 状态 
  var state: Integer = _
  // 名称 
  var name: String = _
  //操作IP 
  @BeanProperty
  var ip: String = _
}

File: admin/src/main/java/com/weicoder/admin/po/LogsOperate.java
package com.weicoder.admin.po;

import javax.persistence.Entity;

import lombok.Getter;
import lombok.Setter;

import org.hibernate.annotations.DynamicInsert;
import org.hibernate.annotations.DynamicUpdate;
import org.springframework.beans.factory.config.ConfigurableBeanFactory;
import org.springframework.context.annotation.Scope;
import org.springframework.stereotype.Component;

import com.weicoder.ssh.entity.EntityIp;
import com.weicoder.ssh.entity.EntityUserId;
import com.weicoder.ssh.entity.base.BaseEntityIdTime;

/**
 * 操作日志实体
 * @author WD
 * @since JDK7
 * @version 1.0
 */
@Component
@Scope(ConfigurableBeanFactory.SCOPE_PROTOTYPE)
@Entity
@DynamicInsert
@DynamicUpdate
@Getter
@Setter
public class LogsOperate extends BaseEntityIdTime implements EntityUserId, EntityIp {
	// 内容
	private String	content;
	// 用户ID
	private Long		userId;
	// 状态
	private Integer	state;
	// 名称
	private String	name;
	// 操作IP
	private String	ip;
}